### ISSUE TITLE
Confusion between CITE tests, specification and schemas published online

### ISSUE THREAD

**Issue by aaime** [opengeospatial/ets-wcs20#145]
Some extensions declare different namespaces in the PDF specification (followed by CITE tests) and in the published schemas.

For example, interpolation:

* In the specification, page 1 (these are not XML namespaces, but it's easy to confuse them):

> This document establishes the following requirements and conformance classes:
>  interpolation, of URI http://www.opengis.net/spec/WCS_service-extension_ interpolation/1.0/req/interpolation; the corresponding conformance class is interpolation,
with URI http://www.opengis.net/spec/WCS_serviceextension_interpolation/1.0/conf/interpolation

* In the [specification](https://portal.ogc.org/files/12-049), page 3, Table 2, the "int" prefix is bound to http://www.opengis.net/wcs/interpolation/1.0

* https://schemas.opengis.net/wcs/interpolation/1.0/wcsInt.xsd declares 
   ``xmlns:int="http://www.opengis.net/WCS_service-extension_interpolation/1.0``

* https://schemas.opengis.net/wcs/interpolation/1.1/wcsInt.xsd declares: 
   ``xmlns:int="http://www.opengis.net/WCS_service-extension_interpolation/1.1``

* The [test for interpolation](https://github.com/opengeospatial/ets-wcs20/blob/master/src/main/scripts/ctl/ext_get-int.xml) declares ``xmlns:int="http://www.opengis.net/wcs/interpolation/1.0"``. 

I'm going to guess the schemas at schemas.opengis.org are wrong... could you confirm?

**Comment by dstenger** [comment]
Thank you for reporting.

I also come to the conclusion that the namespace used in the schemas at schemas.opengis.org are wrong.
However, this question should be answered the SWG (CC @pebau). Thus I propose to contact the SWG directly.

**Comment by pebau** [comment]
indeed, quite confusing - not only for you. By way of history, when these specs were written coverages happened to be the first users of a ModSpec in beta (at best) - see how ModSpec discussion is still going on today.

I always had asked for a recipe "OGC URLs for dummies", so suitable for me.  We should look at that as the ground truth for inspection, verification, and - should it be necessary - fixing.

@dstenger, @ghobona : can you point us to the authoritative OGC URL reference? Based on that, I am available for getting hands URL-dirty.

**Comment by pebau** [comment]
> This document establishes the following requirements and conformance classes:
> interpolation, of URI http://www.opengis.net/spec/WCS_service-extension_ interpolation/1.0/req/interpolation; the corresponding conformance class is interpolation,
> with URI http://www.opengis.net/spec/WCS_serviceextension_interpolation/1.0/conf/interpolation

just checked: in the document I find uniformly "...service-extension..."  (hyphen between service and extension) BUT indeed a space after the underscore ("..._ ..."), in the conformance clause.

@ghobona How can we best handle this? simply update 24-018r1 (given it is minimal), or a corrigendum?

**Comment by pebau** [comment]
re "int" prefix: a simple (and unsatisfying) answer is in Clause 5.4: "When multiple representations of the same information are given in an OGC Standard these are consistent. Should this not be the case then this is considered an error, and the XML schema SHALL take precedence."

The versions actually are correct:

- https://schemas.opengis.net/wcs/interpolation/1.0/wcsInt.xsd declares
xmlns:int="http://www.opengis.net/WCS_service-extension_interpolation/1.0
- https://schemas.opengis.net/wcs/interpolation/1.1/wcsInt.xsd declares:
xmlns:int="http://www.opengis.net/WCS_service-extension_interpolation/1.1

CITE seems to not support 1.1 yet, so it says 1.0.

PS: yes, this naming jungle was a headache back in 2012 when modular specs were uncharted land, and it remains a headache today.
